CMake/fletch-tarballs.cmake

```diff
@@ -121,24 +121,30 @@ if ( NOT fletch_ENABLE_OpenCV OR OpenCV_SELECT_VERSION VERSION_LESS 3.0.0 )
   unset(fletch_ENABLE_OpenCV_contrib CACHE)
 endif()
 
-if (OpenCV_SELECT_VERSION VERSION_EQUAL 3.1.0)
-  set(OpenCV_version "3.1.0")
+if (fletch_ENABLE_OpenCV OR fletch_ENABLE_ALL_PACKAGES)
+  set(OpenCV_version ${OpenCV_SELECT_VERSION})
```

Comment:
...err, actually it seems like the logic for disabling `fletch_ENABLE_OpenCV_contrib` based on opencv settings makes this less trivial change. Maybe I can rework it.